lms/djangoapps/grades/admin.py

```diff
@@ -0,0 +1,27 @@
+"""
```

Comment:
Why move all the admin code back to the top-level?  I really liked the folder structure that was created.
If you're having django issues, just have the top-level models.py file import the admin-specific one.